PITCHME.md

```diff
@@ -256,6 +256,18 @@ A categorized list of records
 
 ---
 
+# Indirect associations
```

Comment:
Can you change "indirect associations" to "many-to-many relationship"? That way we are consistent.